files/update.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
These files belong under `/files/startbootstrap-agency` not directly under files